api/src/main/java/org/hisp/dhis/android/dashboard/api/utils/DbUtils.java

```diff
@@ -94,14 +94,11 @@ public void run() {
                 continue;
             }
 
-            // if the last updated field in up to date model is after the same
```

Comment:
If this code still there, only saves the dashboard items that have the updated date before the last updated date. The problem with this is when I update the order of all the dashboard items of a dashboard when reach to here it only saves da order of the new item added. For example if I have 3 items with order 0,1,2 then I add a new one I put the order to the items 0,1,2,3 but then when comes to here only saves the new one, the position saved will be 0,0,1,2.